PGM/src/main/java/tc/oc/pgm/commands/MapCommands.java

```diff
@@ -262,20 +344,22 @@ public static void mapnext(CommandContext args, CommandSender sender) throws Com
         sender.sendMessage(ChatColor.DARK_PURPLE + PGMTranslations.get().t("command.map.next.success", sender, next.getInfo().getShortDescription(sender) + ChatColor.DARK_PURPLE));
     }
 
-    private static @Nullable BaseComponent formatContribution(Contributor contributor) {
+    private static @Nullable
+    BaseComponent formatContribution(Contributor contributor) {
         BaseComponent c = formatContributor(contributor);
-        if(c == null || !contributor.hasContribution()) return c;
+        if (c == null || !contributor.hasContribution()) return c;
         return new Component(c, new Component(ChatColor.GRAY, ChatColor.ITALIC).extra(" - ").extra(contributor.getContribution()));
     }
 
-    private static @Nullable BaseComponent formatContributor(Contributor contributor) {
+    private static @Nullable
+    BaseComponent formatContributor(Contributor contributor) {
         Identity identity = contributor.getIdentity();
-        if(identity != null) {
+        if (identity != null) {
             return new PlayerComponent(identity, NameStyle.MAPMAKER);
         }
 
         String name = contributor.getName();
-        if(name != null) {
+        if (name != null) {
```

Comment:
All the changes where you see a space added or an extra new line need to be fixed before merge.